docs/project/roadmap.md

```diff
@@ -45,19 +45,24 @@ implementation work.
 
 ## Key results in 2024
 
-### Carbon's toolchain implements the language enough to support interop with C++
+### Carbon's toolchain implements enough of the language to build realistic code
+
+This goal is not necessarily about complete support for the entire language
+design, but rather enough of it to support building the realistic and
+interesting Carbon code that interoperates with C++. This does include
+prerequisites like the prelude.
 
 Some example language features that we think are key to success here but far
 from an exhaustive list:
 
 -   Imports
```

Comment:
```suggestion
interesting Carbon code that interoperates with C++.

Some example language features that we think are key to success here but far
from an exhaustive list:

-   Imports and prelude
```

Reply:
Sure -- had to manually apply this as Github didn't give me a button and added a couple of words.